content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -73,16 +73,34 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
 
 - **maxSkew** describes the degree to which Pods may be unevenly distributed.
   It must be greater than zero. Its semantics differs according to the value of `whenUnsatisfiable`:
+
   - when `whenUnsatisfiable` equals to "DoNotSchedule", `maxSkew` is the maximum
     permitted difference between the number of matching pods in the target
     topology and the global minimum
-    (the minimum number of pods that match the label selector in a topology domain. For example, if you have 3 zones with 0, 2 and 3 matching pods respectively, The global minimum is 0).
+    (the minimum number of pods that match the label selector in a topology domain.
+    For example, if you have 3 zones with 0, 2 and 3 matching pods respectively,
+    The global minimum is 0).
   - when `whenUnsatisfiable` equals to "ScheduleAnyway", scheduler gives higher
     precedence to topologies that would help reduce the skew.
+
+- **minDomains** indicates a minimum number of eligible domains.
+  When the number of eligible domains with match topology keys is less than `minDomains`,
```

Comment:
Describe what "eligible domains" is in somewhere.
https://github.com/kubernetes/kubernetes/blob/475f7af1c1ba17dd3787c609b8202696629b68c6/pkg/apis/core/types.go#L5629-L5630